activities/Moon.activity/index.html

```diff
@@ -2,52 +2,53 @@
 <html>
 
 <head>
-<meta charset="utf-8" />
-<title>Moon</title>
-<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-<link rel="stylesheet" media="not screen and (device-width: 1200px) and (device-height: 900px)"
-	href="lib/sugar-web/graphics/css/sugar-96dpi.css">
-<link rel="stylesheet" media="screen and (device-width: 1200px) and (device-height: 900px)"
-	href="lib/sugar-web/graphics/css/sugar-200dpi.css">
-<link rel="stylesheet" href="css/activity.css">
+  <meta charset="utf-8" />
```

Comment:
Do not update indentation of the file.
I think the `libnotify.css` file is missing.

Reply:
ok will check